apps/spark/src/main/java/com/linkedin/openhouse/jobs/client/TablesClient.java

```diff
@@ -291,7 +295,14 @@ protected TableMetadata mapTableResponseToTableMetadata(GetTableResponseBody res
             .dbName(responseBody.getDatabaseId())
             .tableName(responseBody.getTableId())
             .build();
-    String creator = getTable(metadata).getTableCreator();
+    GetTableResponseBody tableResponseBody = getTable(metadata);
+
+    if (tableResponseBody == null) {
+      log.error("Error while fetching metadata for table: {}", metadata);
```

Comment:
Should this be just db.table instead of whole metadata?

Reply:
Metadata is only db and table here